test/run_test.go

```diff
@@ -140,7 +140,11 @@ func ExecAt(host string, args ...string) (stdout, stderr []byte, e error) {
 // WARNING: `input` can contain secret data.  Never output `input` to console.
 func ExecAtWithInput(host string, input []byte, args ...string) (stdout, stderr []byte, e error) {
 	agent := sshClients[host]
-	return doExec(agent, input, args...)
+	stdout, stderr, err := doExec(agent, input, args...)
+	if err != nil {
+		e = fmt.Errorf("Exec failed: args: %v, err: %w", args, err)
+	}
+	return
```

Comment:
This does not return `stdout` and `stderr` because line 143 introduces new variables.
My suggestion:
```suggestion
	stdout, stderr, e = doExec(agent, input, args...)
	if e != nil {
		e = fmt.Errorf("Exec failed: args: %v, err: %w", args, e)
	}
	return
```

Reply:
I was wrong.

Reply:
According to CI result, the assignment statement does not hide the named return variable.